src/Metadata/GraphQl/DeleteMutation.php

```diff
@@ -70,7 +71,8 @@ public function __construct(
         ?string $name = null,
         $provider = null,
         $processor = null,
-        array $extraProperties = []
+        array $extraProperties = [],
+        protected ?PersistenceMeansInterface $persistenceMeans = null,
```

Comment:
consistency with other changes, why are you not using constructor promotion as you did in the Query object?